<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_definir_valor</class>
 <widget class="QDialog" name="tela_definir_valor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>202</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>202</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Valores</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>192</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Produto</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>176</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Defina o valor de venda, o custo
e quantidade comprada. </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valorvenda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Valor de venda:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_custo_medio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_quantidade">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>0000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valor_compra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>38</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Custo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_quantidade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Quantidade:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_valor_venda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_confirmar">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>130</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/1345902916_camera_test.png</normaloff>:/img/img/1345902916_camera_test.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>130</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/1345902747_button_cancel.png</normaloff>:/img/img/1345902747_button_cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_valor_venda</tabstop>
  <tabstop>le_custo_medio</tabstop>
  <tabstop>le_quantidade</tabstop>
  <tabstop>btn_confirmar</tabstop>
  <tabstop>btn_cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
